src/pages/ReimbursementAccount/CompanyStep.js

```diff
@@ -80,58 +80,107 @@ class CompanyStep extends React.Component {
             'password',
             'companyPhone',
         ];
+
+        // Keys in this.errorTranslationKeys are associated to inputs, they are a subset of the keys found in this.state
```

Comment:
NAB: I read this comment a few times and couldn't understand the purpose until I checked for other usages in the code.

Reply:
I changed it, is it better or worse? :P